### PR TITLE
Publish minutes of 2023-09-11 TPAC meeting

### PR DESCRIPTION
**Pull request by Rob--W**
Generated from https://docs.google.com/document/d/1QkwhEMtMS67JBUkl_WVPZ4lRSKoWcQNlLJSf_GwSXg8/edit using the tool and process from https://github.com/w3c/webextensions/pull/105.

During this meeting we discussed or mentioned issues from the agenda in #385, including #441, #208, #365, #317.

### CODE REVIEW

**Review by oliverdunk — APPROVED** [review]
LGTM - thanks for the work on this as always!

File: _minutes/2023-09-11-wecg-tpac.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+ * [patrick] There's not a secret way to test extensions in the different browsers, so I think it will require engineering to make it work.  I think we should make it work in WebDriver with whatever hooks we need to leverage WPT.
+ * [simeon] I believe when people test their extensions, they use commandline flags to load the extension as part of the webdriver config.  Are you describing a runtime API in WebDriver for loading / unloading extensions?
+ * [patrick] WPT ultimately wraps around WebDriver. In theory we could standardize the flags, but that is not necessary as wpt abstracts that.
+ * [simeon] Limitation there as I understand it is that the extension is tied to the WebDriver's session lifetime. Are you saying that we should be able to load and unload extensions during a WebDrive session?
+ * [patrick] I don't have a strong opinion.  But I think we should invest the engineering hours in the WebDriver / WPT area.
+ * [simeon] My understanding of wpt is that individual tests are in a single file. If I translate that to extensions, then (…)
```
Comment (Rob--W) [inline _minutes/2023-09-11-wecg-tpac.md]: @dotproto Do you recall what was said? If not, I'll leave it at this.

File: _minutes/2023-09-11-wecg-tpac.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+ 8. Simeon Vincent (Unaffiliated)
+ 9. Carlos Jeurissen (Jeurissen Apps)
+ 10. Tomislav Jovanovic (Mozilla)
+ 11. Maxim Tsoy (DuckDuckGo)
+ 12. Richard Worth (Capital One)
+ 13. Andrea (Igalia)
+ 14. Mukul Purohit (Microsoft)
+ 15. Patrick Kettner (Google Chrome)
+ 16. David Johnson (Apple)
+
```
Comment (xeenon) [inline _minutes/2023-09-11-wecg-tpac.md:42]: Appologies, I forgot to sign in. Can you add "17. Timothy Hatcher (Apple)"?

Thanks!

**Review by xeenon — APPROVED** [review]

**Comment by Rob--W** [comment]
Thanks for the reviews. I'll merge it because we have another meeting today (in 5 hours).

**Pull request merged**